File: Tp POO/Ejercicio3/Ingrediente.py
class Ingrediente:
    def __init__(self, nombre, cantidad,unidadMedida):
        self.nome = nombre
        self.quantidade = cantidad
        self.unidadeMedida = unidadMedida
    def __str__(self):
        return f"\t-Nombre: {self.nombre}\n\t-Cantidad: {self.quantidade}\n\t-Unidad de Medida: {self.unidadeMedida}"

File: TPS Prog1/Tps lab Java/src/Asociaciones/Restaurante/Ingrediente.java
package Asociaciones.Restaurante;

public class Ingrediente {
    private String nombre;
    private int cantidad;
    private String unidadMedida;

    public Ingrediente(String nombre, int cantidad, String unidadMedida) {
        this.nombre = nombre;
        this.cantidad = cantidad;
        this.unidadMedida = unidadMedida;
    }

    public String getNombre() {
        return nombre;
    }

    public int getCantidad() {
        return cantidad;
    }

    public String getUnidadMedida() {
        return unidadMedida;
    }

}
